packages/editor-ui/src/views/WorkflowHistory.vue

```diff
@@ -75,12 +107,15 @@ const downloadVersion = async (id: WorkflowVersionId) => {
 		route.params.workflowId,
 		id,
 	);
-	if (workflowVersion?.workflow) {
-		const { workflow } = workflowVersion;
-		const blob = new Blob([JSON.stringify(workflow, null, 2)], {
-			type: 'application/json;charset=utf-8',
-		});
-		saveAs(blob, workflow.name.replace(/[^a-z0-9]/gi, '_') + '.json');
+	if (workflowVersion?.nodes && workflowVersion?.connections && activeWorkflow.value) {
+		const { connections, nodes } = workflowVersion;
+		const blob = new Blob(
+			[JSON.stringify({ ...activeWorkflow.value, nodes, connections }, null, 2)],
+			{
+				type: 'application/json;charset=utf-8',
+			},
+		);
+		saveAs(blob, activeWorkflow.value.name.replace(/[^a-z0-9]/gi, '_') + '.json');
```

Comment:
1. Why are we so strict with overriding non-lowercase and non-digits from the filename? Shouldn't it be fine to allow foreign chars?
2. Should we add the version ID to the downloaded JSON?

Reply:
1. I don't know, just copied this from the other download functionality but I'm fine with replacing just non letter characters to underscores
2. Good idea, I'll add it